config/state_machine/configs/00_moj/40_offender_sar/20_offender_sar_standard_workflow.yml

```diff
@@ -1,137 +1,34 @@
       standard:
-        initial_state: unassigned
+        initial_state: data_to_be_requested
         user_roles:
 
         ########################### SAR :: STANDARD WORKFLOW :: MANAGER ########################
           manager:
             states:
-              unassigned:
+              data_to_be_requested:
+                mark_as_waiting_for_data:
+                  transition_to: waiting_for_data
+                add_message_to_case:
+              waiting_for_data:
+                mark_as_ready_for_vetting:
+                  transition_to: ready_for_vetting
+                add_message_to_case:
+              ready_for_vetting:
+                mark_as_vetting_in_progress:
+                  transition_to: vetting_in_progress
+                add_message_to_case:
+              vetting_in_progress:
+                mark_as_ready_to_copy:
+                  transition_to: ready_to_copy
+                add_message_to_case:
+              ready_to_copy:
+                mark_as_ready_to_dispatch:
+                  transition_to: ready_to_dispatch
+                add_message_to_case:
+              ready_to_dispatch:
+                mark_as_closed:
+                  transition_to: closed
+                add_message_to_case:
+              closed:
```

Comment:
If `case_destroyed` were to be added now would it break everything?

Reply:
I don't think so... what risks are you thinking of here?